### PR TITLE
remove long-deprecated docker config

### PR DESCRIPTION
**Pull request by ryanking**
We long-ago removed this functionality, now remove it from the config
file.

### CODE REVIEW

File: docs/getting-started/tutorial.md [hunk 1]
```diff
@@ -285,7 +283,7 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     A `terraform/staging` directory has been creaded with a few files in it, but nowhere to put terraform files yet– those go in components which are nested in envs. So let's create a component–
 
-1. *create vpc component*
+2. *create vpc component*
```
Comment (cziie) [inline docs/getting-started/tutorial.md:286]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/1/1]

File: docs/getting-started/tutorial.md [hunk 2]
```diff
@@ -381,7 +379,7 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     Fogg has taken care of the drudgery of figuring out what variables and outputs this module supports. All you have left to do is define some `locals` to specify the inputs to the module. Create a `locals.tf` file in that same directory defining the values we need and we're good to go.
 
-1. *create database component*
+3. *create database component*
```
Comment (cziie) [inline docs/getting-started/tutorial.md:382]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 3; Style: 1/1/1]

File: docs/getting-started/tutorial.md [hunk 3]
```diff
@@ -464,7 +462,7 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     Note that since we didn't specify a module_source here, the main.tf file in the database component is empty, Fogg is just creating the scaffolding, not any infrastructure for the database. You can then edit that main.tf file to create the infrastructure you want in that component.
 
-1. create server component
+4. create server component
```
Comment (cziie) [inline docs/getting-started/tutorial.md:465]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 4; Style: 1/1/1]

File: docs/getting-started/tutorial.md [hunk 4]
```diff
@@ -556,6 +554,6 @@ Note that fogg works by generating Terraform and Make files. It does not run any
 
     Now you have separate components for your VPC, database and server, in which you can create infrastructure which is managed by isolated Terraform state files.
 
-1. TODO create prod
-1. TODO refactor to module
-1. TODO create module
+5. TODO create prod
```
Comment (cziie) [inline docs/getting-started/tutorial.md:557]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 5; Style: 1/1/1]

File: docs/getting-started/tutorial.md [hunk 5]
```diff
@@ -556,6 +554,6 @@ Note that fogg works by generating Terraform and Make files. It does not run any
...
+6. TODO refactor to module
```
Comment (cziie) [inline docs/getting-started/tutorial.md:558]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 6; Style: 1/1/1]

File: docs/getting-started/tutorial.md [hunk 6]
```diff
@@ -556,6 +554,6 @@ Note that fogg works by generating Terraform and Make files. It does not run any
...
+7. TODO create module
```
Comment (cziie) [inline docs/getting-started/tutorial.md:559]: **[markdownlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 7; Style: 1/1/1]

**Review by kuannie1 — APPROVED** [review]

**Pull request merged**